src/main/resources/static/api.yaml

```diff
@@ -88,4 +141,88 @@ components:
       properties:
         hipId:
           type: string
-          description: Identifier of the health information provider where patient record needs to be looked up.
\ No newline at end of file
+          description: Identifier of the health information provider where patient record needs to be looked up.
+
+    ConsentRequestsRepresentation:
+      type: object
+      properties:
+        limit:
+          type: integer
+        size:
+          type: integer
+        offset:
+          type: integer
+        requests:
+          type: array
+          maxItems: 100
+          items:
+            $ref: '#/components/schemas/ConsentRequestRepresentation'
+
+    ConsentRequestRepresentation:
```

Comment:
Should have an "id". This is the request Id - This should be the id that was sent to HIU or any CM-Client when the request was placed. NHS doc calls this as txnId. https://app.swaggerhub.com/apis/NCG5/hdaf_api/0.9#/ConsentsRequest